Build failed

When I tried to build StorageDrawers from source I got the following error:

`FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'StorageDrawers-1.10'.
> Could not resolve all dependencies for configuration ':deobfCompile'.
   > Configuration with name 'default' not found.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED
`

Am I doing something wrong?

--- a/README.md
+++ b/README.md
@@ -2,3 +2,24 @@ StorageDrawers
 ==============
 
 A mod adding compartmental storage for Minecraft Forge
+
+For Players
+-----------
+
+StorageDrawers may be downloaded from any of the following sites:
+
+- [Minecraft Forums](http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/2198533-storage-drawers-v1-10-7-v3-5-0-v4-0-0-updated-nov)
+- [Curse.com](https://mods.curse.com/mc-mods/minecraft/223852-storage-drawers)
+- [Github Releases](https://github.com/jaquadro/StorageDrawers/releases)
+
+For Developers
+--------------
+
+StorageDrawers is built using `gradle`. To build StorageDrawers, the support library [Chameleon](https://github.com/jaquadro/Chameleon) must be checked out in an adjacent directory. These commands should be enough to get you started:
+
+```
+git clone https://github.com/jaquadro/Chameleon 
+git clone https://github.com/jaquadro/StorageDrawers
+cd StorageDrawers
+./gradle build
+```

